cmds/core/tee/tee_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestTee(t *testing.T) {
+	tempDir := t.TempDir()
+
+	tests := []struct {
+		input  string
+		args   []string
+		append bool
+	}{
+		{
+			"hello",
+			[]string{filepath.Join(tempDir, "a1"), filepath.Join(tempDir, "a2")},
+			false,
```

Comment:
nit: can you please add the field names here?  It makes it easier to parse what the test case is while reading it, instead of having to go back to the struct definition and see what the 'bool' is.

Reply:
sure